build-tools/packages/version-tools/src/versions.ts

```diff
@@ -19,21 +24,25 @@ export function getSimpleVersion(
 	fileVersion: string,
 	argBuildNum: string,
 	argRelease: boolean,
-	patch: boolean,
+	simplePatchVersioning: boolean,
 ) {
 	// Azure DevOp passes in the build number as $(buildNum).$(buildAttempt).
 	// Get the Build number and ignore the attempt number.
-	const buildId = patch ? parseInt(argBuildNum.split(".")[0], 10) : undefined;
+	const buildId = simplePatchVersioning ? parseInt(argBuildNum.split(".")[0], 10) : undefined;
 	let version = fileVersion;
 	if (isInternalVersionScheme(version, /* allowPrereleases */ true)) {
-		if (patch) {
+		if (simplePatchVersioning) {
 			throw new Error(
 				`Cannot use simple patch versioning with Fluid internal versions. Version: ${fileVersion}`,
 			);
 		}
 
 		if (!argRelease) {
-			version = changePreReleaseIdentifier(version, "dev");
+			const [, , prereleaseId] = fromInternalScheme(version);
+			version = changePreReleaseIdentifier(
+				version,
+				prereleaseId === ALLOWED_PRERELEASE_IDENTIFIERS[0] ? "dev" : `dev-${prereleaseId}`,
+			);
 		}
```

Comment:
Is this an important change? To distinguish dev cases?
If so, can we have dev-rc and dev-internal so that both are clear?

Reply:
I wrote it this way so that it was easier to maintain if the prerelease IDs change (they have changed twice while this
PR has been open). I could change it now, but I'm inclined to leave that for a future change. I'm not 100% convinced we
will ship with "rc" as the final prerelease ID. :)

Reply:
Clarification: I am not objecting to `dev-rc`. I am saying that the default shouldn't be special. `dev-internal` is good.
If you are trying to preserve `internal` -> `dev` then should that always be done? And then it isn't about the default, but about `internal`.
If we never have `internal` again, then why is it a default?
I think it is okay if all of `internal` dev versions become `dev-internal` now. Would be clear. I could also be wrong. ;)

Reply:
Churning the version schemas has proven to be error-prone, and I don't want to unintentionally break anything that
relies on the existing dev versions.

Reply:
That is cool. This intent (as I understand it) doesn't match the code. You are saying that `internal` dev should remain being `dev`. So let's make that explicit and add a comment.
(It doesn't really have anything to do with DEFAULT except that DEFAULT happens to have the value `internal`.)

Reply:
Ahh, I see. Thanks. Fixed in latest.